bin/oneoff/backfill_data/backfill_script_ids_in_teacher_feedbacks_2.rb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+# Backfill existing TeacherFeedbacks to set script_id based on script_level_id.
+
+require 'optparse'
+
+$options = {actually_update: false}
+OptionParser.new do |opts|
+  opts.banner = "Usage: #{File.basename(__FILE__)} [options]"
+  opts.on('-u', '--actually-update', 'Actually perform the update.') do
+    $options[:actually_update] = true
+  end
+  opts.on('-h', '--help', 'Add -u to perform the update.') do
+    puts opts
+    exit
+  end
+end.parse!
+puts "Called with options: #{$options}"
+
+require_relative '../../../dashboard/config/environment'
+
+$stdout.sync = true
+
+def feedbacks_without_script_id
+  TeacherFeedback.where(script_id: nil)
```

Comment:
Should we fetch and fix soft-deleted records as well with the `with_deleted` scope?

Reply:
ruh-roh. I should have thought of that in the original backfill script. We may want to run that again  with the `with_deleted` scope.

Reply:
Great catch! Fortunately there are not too many:
```
[production] dashboard > TeacherFeedback.with_deleted.where.not(deleted_at: nil).count
=> 228
```
I've updated this PR to use with_deleted in both backfill scripts, then I'll rerun the first backfill script before running this one.